ci-operator/step-registry/openshift-qe/run-api-apf-customized-flowcontrol/openshift-qe-run-api-apf-customized-flowcontrol-commands.sh

```diff
@@ -6,12 +6,12 @@ set -o pipefail
 ###############################################
 ## Auth=fbledsoe@redhat.com, liqcui@redhat.com
 ## Description: Test building new PriorityLevelConfiguration and FlowSchemas, and queueing and dropping excess requests. 
-## Polarion test case: OCP-41643 - Load cluster to test bad actor resilience	
-## https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-41643
-## Bug related: https://issues.redhat.com/browse/OCPBUGS-12266
-## Cluster config: 3 master (m5.4xlarge or equivalent)
+## Polarion test case: OCP-69945 - Load cluster to test bad actor resilience	
```

Comment:
I saw this script in test case  https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-69945

Reply:
OCP-69945 - Test customization of API Priority and Fairness flow control settings